src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresConnectorControlImpl.java

```diff
@@ -294,14 +219,29 @@ public void showControlPoints() {
 
     @Override
     public void hideControlPoints() {
-        if (m_HandlerRegistrationManager != null) {
-            destroyPointHandles();
-        }
+        destroyPointHandles();
+    }
+
+    @Override
+    public boolean areControlPointsVisible()
+    {
+        return null != m_connector.getPointHandles() && m_connector.getPointHandles().isVisible();
     }
 
     @Override
-    public Point2D adjustControlPointAt(double x, double y, double deltaX, double deltaY) {
-        return m_connector.getLine().adjustPoint(x, y, deltaX, deltaY);
+    public void moveControlPoint(final int index,
```

Comment:
the method return could be a boolean to tell if the move was executed or rolled back.